### DISCUSSION TITLE
Return the normalized form of the query in the `query` field

### DISCUSSION THREAD

**Discussion by xamir82** [meilisearch/product#699] [Feedback & Feature Proposal]
Hey there! Currently the `query` field in the Meilisearch search endpoint's response is exactly equal to the query that was sent in the request.

It would be useful in some cases to know the normalized form of the query based on which Meilisearch actually did the search. For example, if the query is an Arabic word with diacritics, Meilisearch currently [normalizes](https://github.com/meilisearch/charabia/issues/137) the query by removing the diacritics, but it still returns the original query in the `query` field.

There are scenario where you would want to know the normalized form of the query — e.g. for example if you want to then store the (normalized) query in a `popular_queries` collection that will power an auto-suggestion feature, or if you want to display the normalized form of the query on the UI as the title of your site's the search page, and so on.

This could be returned as the existing `query` response field, or a separate `normalizedQuery` field, for instance.

**Comment by curquiza** [comment]
Hello @xamir82 
I move this feature request to the right place. You can see the right place when creating an issue to the meilisearch repo

<img width="1232" alt="Capture d’écran 2023-10-11 à 13 35 59" src="https://github.com/meilisearch/product/assets/20380692/61512ae8-ab40-4f2c-99a2-ac7c258f24b0">

 
This repo (meilisearch) does not take feature requests: we manage them in [our public roadmap](https://roadmap.meilisearch.com/tabs/1-under-consideration), or if you want more interaction, in the [product repository](https://github.com/meilisearch/product/discussions)

Thanks for your feedback

**Reply by xamir82** [reply]
Ah okay, sorry! I'll keep this in mind next time!

**Comment by curquiza** [comment]
Pinging @meilisearch/engine-team to know how doable it is to do

**Reply by Kerollmops** [reply]
It seems feasible to me but maybe @ManyTheFish would be better to answer this question!
